### ISSUE TITLE
Softmax for weight annealing

### ISSUE THREAD

**Issue by rsundar96** [rsundar96/curriculum-learning-acceleration#8]
The following training and test curves were  obtained for the  different Softmax initialisation methods.

### I. Training Loss

<img width="862" alt="Screenshot 2020-09-08 at 11 06 48" src="https://user-images.githubusercontent.com/19652780/92463562-e3bdb980-f1c3-11ea-8e1a-388cffc447d3.png">

<img width="864" alt="Screenshot 2020-09-08 at 11 07 31" src="https://user-images.githubusercontent.com/19652780/92463573-e6201380-f1c3-11ea-8f72-2db7487ea838.png">

### II. Test Loss

<img width="864" alt="Screenshot 2020-09-08 at 11 09 33" src="https://user-images.githubusercontent.com/19652780/92463578-e7514080-f1c3-11ea-9fdd-ef92b6193c08.png">

<img width="864" alt="Screenshot 2020-09-08 at 11 10 14" src="https://user-images.githubusercontent.com/19652780/92463582-e8826d80-f1c3-11ea-8015-f892a944c3b4.png">
